spec/features/user_can_do_things_spec.rb

```diff
@@ -0,0 +1,62 @@
+require 'rails_helper'
+
+describe 'as a user' do
+  describe 'when logged in' do
+    it 'shows things on dashboard' do
+      user_1 = create(:user, github_token: ENV["GITHUB_TOKEN"])
+      token_1 = user_1.github_token
+
+      allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(user_1)
+
+      visit '/dashboard'
+      expect(page).to have_content('Github')
+
+      within('.github-list') do
+        expect(page).to have_css('.repository', count: 5)
+      end
+
+      within('.following-list') do
```

Comment:
The get_json refactor I did removes the 5-person limit from following & followers, but if we don't ultimately merge that code, then sticking with this test expectation will be fine.